OpenQA/Isotovideo/Utils.pm

```diff
@@ -321,6 +321,7 @@ sub handle_generated_assets ($command_handler, $clean_shutdown) {
             my $dir = 'assets_private';
             my $name = $bmwqemu::vars{"STORE_HDD_$i"} || undef;
             unless ($name) {
+                next if $bmwqemu::vars{"PUBLISH_HDD_$i"} =~ /none|None/;
```

Comment:
```suggestion
                next if $bmwqemu::vars{"PUBLISH_HDD_$i"} =~ /none/i;
```

But this is probably the wrong place. It would make much more sense to skip the whole `handle_generated_assets` function early.

Reply:
I'm wondering why it isn't suffiicient to just allow `none`

Reply:
Change id done. I took a look around but i am not sure how easy or small will be to move the logic somewhere else. We can discuss it. For now i am thinking to add a test but my system is broken atm and test do not run.

Reply:
> But this is probably the wrong place. It would make much more sense to skip the whole `handle_generated_assets` function early.

Why would you completely skip all asset uploads if only PUBLISH_HDD_x is none?
In that case I think a special variable `NO_ASSET_UPLOAD` would IMHO make more sense.
But you are more experienced with assets, so maybe I'm missing something.

Reply:
You're right. So let's keep it within that loop.